kebab_city_tour/src/router.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Home from './Home';
+import Map from './Map';
+import LoginPage from './LoginPage';
+
+const AppRouter = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/map" element={<Map />} />
+      <Route path="/login" element={<LoginPage />} />
+    </Routes>
+  </Router>
+);
+
+export default AppRouter;
```

Comment:
sorry for bothering you, but you didn't clean this file :<

Reply:
✔️